docs/deploy_local.md

```diff
@@ -51,6 +51,7 @@ export BOND_MEDIATOR_CONTRACT=${BondMediator}
 export BOND_FACTORY_CONTRACT=${BondFactory}
 export COLLATERAL_TOKENS_CONTRACT=${BitToken}
 export TREASURY_ADDRESS=${Treasury}
+export DAO_METADATA=${metadataString}
```

Comment:
issue: Entry needed in `#### Environment Variables needed for Verify`

You're adding an env var but not adding the description for it.